### PR TITLE
change DCT for Scim

### PR DESCRIPTION
**Pull request by Eebru-gzy**
This PR changes the DCT environment variable to 0 when it is pushing the SCIM image

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -209,6 +209,9 @@ jobs:
           DOCKER_CONTENT_TRUST_REPOSITORY_PASSPHRASE: ${{ steps.setup-dct.outputs.dct-delegate-repo-passphrase }}
           SERVICE_NAME: ${{ steps.setup.outputs.service_name }}
         run: |
+          if [ "$SERVICE_NAME" == "Scim" ]; then
```
Comment (michalchecinski) [inline .github/workflows/release.yml:212]:
```suggestion
          if [ "$SERVICE_NAME" == "scim" ]; then
```

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -209,6 +209,9 @@ jobs:
...
+            $DOCKER_CONTENT_TRUST=0
```
Comment (michalchecinski) [inline .github/workflows/release.yml:213]:
```suggestion
            export DOCKER_CONTENT_TRUST=0 
```

**Review by michalchecinski — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (Eebru-gzy) [reply .github/workflows/release.yml:212]: Did you check the Service name?

File: .github/workflows/release.yml (hunk 2 above)
Comment (Eebru-gzy) [reply .github/workflows/release.yml:213]: Okay

File: .github/workflows/release.yml (hunk 1 above)
Comment (Eebru-gzy) [reply .github/workflows/release.yml:212]: I can see it is changed to lowercase later

**Review by michalchecinski — APPROVED** [review]

**Pull request merged**